melissa/profile_populator.py

```diff
@@ -139,6 +139,17 @@ def empty(variable):
     if len(icloud_username) > 0:
         icloud_password = getpass()
 
+    while(True):
```

Comment:
No he already merge my branch, so it is already tested. 

In overall it is ready to be merged